### PR TITLE
Re-format inputs

### PR DESCRIPTION
**Pull request by AnnaSyme**
Just doing one first as a test

### CODE REVIEW

**Comment by neoformit** [comment]
This looks fine, can you commit the rest here?

File: webapp/home/labs/genome/annotation.yml [hunk 1]
```diff
@@ -124,18 +124,18 @@ tabs:
...
-                  label: RNAseq Illumina reads
-                - datatypes:
+                - label: Alternative annotation
+                  datatypes:
                     - gff3
-                  label: Alternative annotation
-                - datatypes:
-                    - gbk
-                  label: Alternative annotation
+                - label: Alternative annotation
```
Comment (neoformit) [inline webapp/home/labs/genome/annotation.yml:136]: Are there two "Alternative annotation" inputs, or is this one input with two accepted datatypes?

image

**Review by neoformit — COMMENTED** [review]
One comment otherwise looks good

File: webapp/home/labs/genome/annotation.yml (hunk 1 above)
Comment (AnnaSyme) [reply webapp/home/labs/genome/annotation.yml:136]: @neoformit  there are two inputs, they are both alternative annotations

**Pull request merged**